rollup.config.js

```diff
@@ -1,7 +1,17 @@
 import { nodeResolve } from '@rollup/plugin-node-resolve';
 import commonjs from '@rollup/plugin-commonjs';
 
+const inputFile = process.env.ROLLUP_ENTRYPOINT;
+const outputFile = process.env.ROLLUP_OUT_FILE;
+
 export default {
   context: "window",
+  input: inputFile,
+  output: [
+    {
+      file: outputFile,
+      format: 'esm'
+    }
+  ],
```

Comment:
I don't understand why the changes to this file are needed now and weren't needed before. I trust this makes sense, but don't have confidence in my review of this file.